test/functional/sqlsrv/sqlsrv_574_next_result.phpt

```diff
@@ -0,0 +1,186 @@
+--TEST--
+GitHub issue 574 - Fetch Next Result Test
```

Comment:
please change this test as well according to the comments in the PDO_SQLSRV equivalent test